cmd/assessment/benchmarks/erc20Benchmark.go

```diff
@@ -0,0 +1,72 @@
+package benchmarks
```

Comment:
this file is pretty similar to delegationBenchmark

Reply:
I know, but calls a different method with different parameters. The common part of the both structs is just minor boiler plate code (Name and IsIntefrace on-liner methods. Even the Name methods displays the information in different ways).

Reply:
extracted some common functionality such as create test gas map